server/documents/modules/dropdown.html.eco

```diff
@@ -1341,6 +1341,21 @@ themes      : ['Default', 'GitHub', 'Material']
       </div>
     </div>
 
+    <div class="dropdown example">
+      <h4 class="ui header">Height</h4>
+      <p>A selection dropdown can be higher (2x of default height) so that more items can be glanced.</p>
+      <select class="ui search higher column dropdown" multiple>
```

Comment:
`column` is not needed here

Reply:
```suggestion
      <select class="ui search higher dropdown" multiple>
```

Reply:
Fixed in https://github.com/fomantic/Fomantic-UI-Docs/pull/109/commits/82adb7844427a1ebf9ac1050bb302d3a39e98383